modules/observe/proc-admin-enabling-metrics-ocp-operator.adoc

```diff
@@ -25,24 +25,33 @@ Currently, the {product} Operator does not support creating a `ServiceMonitor` c
 apiVersion: monitoring.coreos.com/v1
 kind: ServiceMonitor
 metadata:
-  name: <custom_resource_name> # <1>
-  namespace: {my-product-namespace} # <2>
+  name: developer-hub-service-monitor <1>
+  namespace: {my-product-namespace} <2>
   labels:
-    app.kubernetes.io/instance: <custom_resource_name>
-    app.kubernetes.io/name: backstage
+    app.kubernetes.io/instance: {my-product-cr-name} <3>
+    app.kubernetes.io/name: {product-custom-resource-type}
 spec:
   namespaceSelector:
     matchNames:
-      - {my-product-namespace}
+      - {my-product-namespace} <4>
   selector:
     matchLabels:
-      rhdh.redhat.com/app: backstage-<custom_resource_name>
+      app.kubernetes.io/instance: developer-hub <5>
+      app.kubernetes.io/name: {product-custom-resource-type}
   endpoints:
   - port: http-metrics
     path: '/metrics'
 ----
-<1> Replace `<custom_resource_name>` with the name of your {product} CR.
-<2> Replace `<project_name>` with the name of the {ocp-short} project where your {product} instance is running.
+<1> The name of your `ServiceMonitor` resource, for example, `developer-hub-service-monitor`.
+<2> The namespace where your `ServiceMonitor` lives.
```

Comment:
Changing the tense here might help clarify that we're setting the namespace where this CR will be. 

i.e. `<2> The namespace where your ServiceMonitor will live.`